lib/utility.js

```diff
@@ -66,28 +66,18 @@ export async function loadData(id) {
 	}
 }
 
-export function addWatchPaths(obj) {
-	const paths = globalThis.VITEPRESS_CONFIG.userConfig.themeConfig.dovecot?.watch_paths
+export function dovecotSetting(name) {
+	const setting = globalThis.VITEPRESS_CONFIG.userConfig.themeConfig.dovecot?.[name]
 
-	if (!paths) {
-		throw new Error("Missing 'watch_paths' setting")
+	if (setting === undefined) {
+		throw new Error("Missing '" + name + "' setting")
 	}
 
-	return { ...obj, ...{ watch: paths } }
-}
-
-export async function manFiles() {
-	/* Check for config override file. */
-	const lconf = await loadLocalConf()
-
-	return lconf?.man_paths ??
-		[ 'docs/core/man/*.[[:digit:]].md' ]
+	return setting
 }
 
-export async function manIncludes() {
-	const lconf = await loadLocalConf()
-	return lconf?.man_includes ??
-		[ 'docs/core/man/include/*.inc' ]
+export function addWatchPaths(obj) {
+	return { ...obj, ...{ watch: dovecotSetting('watch_paths') } }
```

Comment:
I feel like this commit contains two separate commits:
1. add `dovecotSetting()` resolver, and
2. move man-paths out of local config.

Reply:
Then you could do the `dovecotSetting()` commit first, and do not need to change the `addWatchPaths()` after introducing it.